src/types/entities/attachment.rs

```diff
@@ -13,11 +15,13 @@ pub struct Attachment {
     pub proxy_url: String,
     pub height: Option<u64>,
     pub width: Option<u64>,
-    pub message_id: Snowflake,
+    #[cfg_attr(feature = "sqlx", sqlx(skip))]
```

Comment:
This leaves me scratching my head a little, why is the `message_id` replaced by a `message: Option<Message>`?
I can't find either of these two attributes in the documentation for the Attachment object - is this a server-only thing? If so, why is this here? /nm /genq :D I would like to see all server-only attributes being moved to the symfonia, as an extension of the structs defined in chorus, if this is possible.

Reply:
Oh pff, I missed this somehow.  I'd already moved the message_id to it's private wrapper variant in symfonia.